scripts/ccache-clang++.sh

```diff
@@ -1,11 +1,10 @@
 #!/bin/sh
-if type -p /usr/local/bin/ccache >/dev/null 2>&1; then
+if command -v ccache &> /dev/null; then
   export CCACHE_MAXSIZE=10G
   export CCACHE_CPP2=true
   export CCACHE_HARDLINK=true
   export CCACHE_SLOPPINESS=file_macro,time_macros,include_file_mtime,include_file_ctime,file_stat_matches
-  exec /usr/local/bin/ccache /usr/bin/clang++ "$@"
+  exec ccache /usr/bin/clang++ "$@"
```

Comment:
```suggestion
  exec ccache clang++ "$@"
```

I wonder if clang++ could also just be read from the path?

Reply:
As long as it's in the path...which it should be.